numpy/core/src/common/npy_dlpack.h

```diff
@@ -0,0 +1,41 @@
+#include "Python.h"
+#include "dlpack/dlpack.h"
+
+#ifndef NPY_DLPACK_H
+#define NPY_DLPACK_H
+
+// Part of the Array API specification.
+#define NPY_DLPACK_CAPSULE_NAME "dltensor"
+#define NPY_DLPACK_USED_CAPSULE_NAME "used_dltensor"
+
+// Used internally by NumPy to store a base object
+// as it has to release a reference to the original
+// capsule.
+#define NPY_DLPACK_INTERNAL_CAPSULE_NAME "numpy_dltensor"
+
+/* This is exactly as mandated by dlpack */
```

Comment:
I put a comment at https://github.com/dmlc/dlpack/issues/51#issuecomment-849794854 suggesting that dlpack suggest this exact implementation.